pkg/core/blockchain.go

```diff
@@ -1760,7 +1757,7 @@ func (bc *Blockchain) storeBlock(block *block.Block, txpool *mempool.Pool) error
 				stop = start + 1
 			}
 			for index := start; index < stop; index++ {
-				ts, err := kvcache.DeleteBlock(bc.GetHeaderHash(index), bc.config.Ledger.RemoveUntraceableHeaders)
+				ts, err := kvcache.DeleteBlock(bc.GetHeaderHash(index), bc.config.Ledger.RemoveUntraceableBlocks)
```

Comment:
Does this flag make any sense at all now? Do we `DeleteBlock(index, false)` anywhere? Looks like it was added in 9599fba24f4f37537e5602eb37c48107ac3da5c9 and can be dropped now.

Reply:
It's not the latest changes, locally I've removed this flag, but didn't push an update since work on #3795 is still in progress.